### PR TITLE
Update UG with expense deletion commands

### PR DESCRIPTION
**Pull request by sp4ce-cowboy**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -128,6 +128,106 @@ Command Options:
 > Attribute “amount” must be a number (received: 5.80.)
 > ```
 
+
+### Delete Expense
```
Comment (woojiahao) [inline docs/UserGuide.md:132]: Maybe leave a `[coming soon]` just in case? Very minor nit though

File: docs/UserGuide.md [hunk 2]
```diff
@@ -128,6 +128,106 @@ Command Options:
...
+>**Case**:  Delete expense command entered with no matching expense name
+>
+>**Input**: `delete asdf`
+>
+>**Output**: No expense deleted. Delete command must be followed with a valid expense name.
+>
+>**Remark**: No expenses will be removed and no changes made to file.
+
+
+### Mass Delete Expense
```
Comment (woojiahao) [inline docs/UserGuide.md:178]: The `[coming soon]` nit

File: docs/UserGuide.md [hunk 3]
```diff
@@ -128,6 +128,106 @@ Command Options:
...
+
+##### Successful Execution
+
+###### Example 1
+
+>**Case**:  Delete expense named “friday mcdonalds”
+>
+>**Input**: `delete "friday mcdonalds"`
+>
+>**Output**: Successfully deleted expense “friday mcdonalds”!
```
Comment (woojiahao) [inline docs/UserGuide.md:151]: Shall we standardize all outputs to be wrapped in the fenced code block? I saw that @elhy1999 did it so I followed

**Review by woojiahao — CHANGES_REQUESTED** [review]
Minor changes to formatting but other than that, LGTM!

**Comment by sp4ce-cowboy** [comment]
@woojiahao All changes addressed.

**Comment by woojiahao** [comment]
The fenced code blocks is like 

```md
> **Output**:
> ```
> output here
> ```
```

**Comment by sp4ce-cowboy** [comment]
> The fenced code blocks is like
> 
> ```
> > **Output**:
> > ```
> > output here
> > ```
> ```

Done.

**Review by woojiahao — APPROVED** [review]
LGTM

**Pull request merged**